packages/core/src/StarknetWindowObject.ts

```diff
@@ -1,10 +1,55 @@
-import type { AccountInterface, ProviderInterface } from "starknet"
-import type {
-  AccountInterface as AccountInterfaceV4,
-  ProviderInterface as ProviderInterfaceV4,
-} from "starknet4"
+type FELT = string
 
-export type AccountChangeEventHandler = (accounts: string[]) => void
+type Call = {
+  contract_address: FELT
+  entrypoint: string
+  calldata?: FELT[]
+}
+
+type SIERRA_ENTRY_POINT = {
+  selector: FELT
+  function_idx: number
+}
+
+type StarkNetMerkleType = {
```

Comment:
We have a PR to change to `Starknet`
https://github.com/starknet-io/starknet.js/pull/723
Is it possible to do it now, wdyt?

Reply:
done